charts/dawarich/templates/deployment.yaml

```
@@ -31,7 +31,27 @@ spec:
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       initContainers:
-        {{ include "dawarich.initContainers" . | nindent 10 }}
+        {{ include "dawarich.initContainers" . | nindent 8 }}
+        - name: postgis-init
+          image: bitnami/postgresql:latest
+          command:
+            - /bin/sh
+            - -c
+            - |
+              set -e
+              psql -c "ALTER USER ${DB_USER} WITH SUPERUSER;"
+          env:
+            - name: PGHOST
+              value: {{ include "dawarich.fullname" . }}-postgresql
+            - name: PGUSER
+              value: postgres
+            - name: PGPASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "dawarich.fullname" . }}-postgresql
+                  key: postgres-password
```

```suggestion
              valueFrom:
                secretKeyRef:
                  {{- if .Values.postgresql.auth.existingSecret }}
                  name: {{ include "dawarich.fullname" . }}-postgresql
                  {{- else }}
                  name: {{ .Values.postgresql.auth.existingSecret }}
                  {{- end }}
                  key: postgres-password
```

To make it work with an existing secret for postgresql :)

---

Added in 1596ad1